pkg/apiserver/rest/apis/v1/types.go

```diff
@@ -346,7 +346,7 @@ type ApplicationStatusResponse struct {
 type ApplicationStatisticsResponse struct {
 	EnvCount      int64 `json:"envCount"`
 	TargetCount   int64 `json:"targetCount"`
-	RevisonCount  int64 `json:"revisonCount"`
+	RevisionCount int64 `json:"revisionCount"`
```

Comment:
will it cause incompatible？

Reply:
The VelaUX has changed.

Reply:
Does velaUX addon set the system version requirement?